src/main/java/core/basesyntax/Demo.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Demo {
+    static final int sizeOfArrayForFigures = 5;
```

Comment:
```suggestion
     private static final int MAX_ARR_SIZE = 5;
```
don't forget about access modifiers + we use upper case to declare constant